### PR TITLE
Adding CIT LIGO's OSDF Stashcache (pull request attempt #2)

### PR DESCRIPTION
**Pull request by dankozak**
Adding CIT LIGO's OSDF Stashcache (pull request attempt #2)

### CODE REVIEW

File: topology/California Institute of Technology/Caltech LIGO/CIT_LIGO.yaml [hunk 1]
```diff
@@ -110,4 +110,21 @@ Resources:
...
+      Administrative Contact:
+        Primary:
+          Name: Stuart Anderson
+          ID: c50e7cc9d0086272ef995fb76461612d40c70435
+      Security Contact:
+        Primary:
+          Name: Stuart Anderson
+          ID: c50e7cc9d0086272ef995fb76461612d40c70435
+    FQDN: stashcache.ligo.caltech.edu
+    ID:
```
Comment (mwestphall) [inline topology/California Institute of Technology/Caltech LIGO/CIT_LIGO.yaml:124]:
```suggestion
    ID: 1461
```

File: topology/California Institute of Technology/Caltech LIGO/CIT_LIGO.yaml [hunk 2]
```diff
@@ -110,4 +110,21 @@ Resources:
...
+          Name: Stuart Anderson
+          ID: c50e7cc9d0086272ef995fb76461612d40c70435
+      Security Contact:
+        Primary:
+          Name: Stuart Anderson
+          ID: c50e7cc9d0086272ef995fb76461612d40c70435
+    FQDN: stashcache.ligo.caltech.edu
+    ID: 
+    Services:
+      Squid:
```
Comment (mwestphall) [inline topology/California Institute of Technology/Caltech LIGO/CIT_LIGO.yaml:126]:
```suggestion
      XRootD cache server:
```

**Review by mwestphall — CHANGES_REQUESTED** [review]

File: topology/California Institute of Technology/Caltech LIGO/CIT_LIGO.yaml [hunk 3]
```diff
@@ -110,4 +110,21 @@ Resources:
         Description: Frontier-squid proxy server
     AllowedVOs:
       - LIGO
+  CIT_LIGO_STASHCACHE:
```
Comment (mwestphall) [inline topology/California Institute of Technology/Caltech LIGO/CIT_LIGO.yaml:113]:
```suggestion
  CIT_LIGO_STASHCACHE:
    Active: true  
```

**Comment by mwestphall** [comment]
@dankozak Would you be able to provide a DN field for the cache? It should look something like:
```
  CIT_LIGO_STASHCACHE:
    ...
    DN: /DC=org/DC=opensciencegrid/O=Open Science Grid/OU=Services/CN=my-auth-cache.example.net
    ...
```

**Comment by mwestphall** [comment]
@dankozak Sorry, one last thing. Would you be able to merge the mainline master branch back into your branch? Several tests have been updated

**Comment by dankozak** [comment]
> @dankozak Sorry, one last thing. Would you be able to merge the mainline master branch back into your branch? Several tests have been updated

I think I have now done so, but I will admit I find Github to be completely obtuse so I'm never sure where I am or what's happening. :-)

**Review by mwestphall — APPROVED** [review]

**Pull request merged**